docs/src/test/java/jdbi/doc/CallTest.java

```diff
@@ -24,11 +23,16 @@ public void testCall() {
 
         // tag::invokeProcedure[]
         OutParameters result = handle
-                .createCall("{:sum = call add(:a, :b)}") // <1>
-                .bind("a", 13) // <2>
-                .bind("b", 9) // <2>
-                .registerOutParameter("sum", Types.INTEGER) // <3> <4>
-                .invoke(); // <5>
+            // <1>
```

Comment:
Does making these changes break the asciidoc?

Reply:
will undo